docs/how-to/container.md

```diff
@@ -0,0 +1,125 @@
+    ```
+
+=== "Apptainer/Singularity"
+
+    ```
+    apptainer run dcm2bids.sif --help
+    ```
+
+## Step 3: Run `dcm2bids_scaffold`
+
+To run `dcm2bids_scaffold` with Apptainer/Singularity, you need to *execute* a command instead of *running* the pre-specified command (`dcm2bids`). You need to bind the respective volumes.
+
+=== "Docker"
+
+    ```
+    docker run --rm -it \
+    --entrypoint /venv/bin/dcm2bids_scaffold \
+    -v /path/to/output-dir:/output \
+    unfmontreal/dcm2bids:latest -o /output/new_bids_dataset
+    ```
+
+=== "Apptainer/Singularity"
+
+    ```
+    singularity exec \
+    -B /path/to/output-dir:/output \
+    dcm2bids.sif dcm2bids_scaffold -o /output/new_bids_dataset
+    ```
+
+
+## Step 4: Run `dcm2bids_helper`
+
+To run `dcm2bids_helper` with Apptainer/Singularity, you need to *execute* a command instead of *running* the pre-specified command (`dcm2bids`). To bind the respective volumes, you have two options:
+
+1. Put the input data in the same parent directory as the output directory.
+2. Specify the input data directory as a separate volume.
+
+If you bind the newly scaffolded directory on its own, you can simply use the `-o /output` instead of having to specify the full path to the scaffolded directory. Same goes for the input data directory, if the input data directory is one subject, you can bind it directly to `/input`. If it is the parent directory of multiple subjects, you can bind it to `/input` and specify the specific subject directory (e.g, `-d /input/subject-01`).
+
+=== "Docker"
+
+    ```
+    docker run --rm -it --entrypoint /venv/bin/dcm2bids_helper \
+    -v /path/to/input-data:/input \
+    -v /path/to/output-dir/new_bids_dataset:/output \
+    unfmontreal/dcm2bids:latest -o /output -d /input
+    ```
+
+=== "Apptainer/Singularity"
+
+    ```
+    singularity exec \
+    -B /path/to/input-data:/input \
+    -B /path/to/output-dir/new_bids_dataset:/output \
+    dcm2bids.sif dcm2bids_helper -o /output -d /input
+    ```
+
+## Step 5: Run `dcm2bids`
+
+You can use `run` as in Step 2 or use `exec dcm2bids` to run `dcm2bids` with the appropriate arguments and options. You need to bind the respective volumes.
+
+You can put input data in the same parent directory as the output directory, or you can specify the input data directory as a separate volume. You must also specify the path to the configuration file. If you use the scaffolded dataset, the config file is usually in the `code/` directory.
+
+You can also [deface your data](use-advanced-commands.md#post_op) and [validate your BIDS data](use-advanced-commands.md#-bids_validate) using the `--bids_validate` flag.
+
+=== "Docker"
+
+    ```
+    docker run --rm -it \
+    -B /path/to/input-data:/input \
+    -B /path/to/output-dir/new_bids_dataset:/output \
+    unfmontreal/dcm2bids:latest --auto_extract_entities --bids_validate \
+    -o /output -d /input -c /output/code/config.json -p 001
+    ```
```

Comment:
```suggestion
    ```
    docker run --rm -it \
    -v /path/to/dicoms:/bids:ro \
    -v /path/to/config.json:/config.json:ro \
    -v /path/to/bids:/bids \
    unfmontreal/dcm2bids:${VERSION} --auto_extract_entities --bids_validate \
    -o /bids -d /dicoms -c /config.json -p 001
    ```
```